airbyte-integrations/connectors/source-bamboo-hr/source_bamboo_hr/manifest.yaml

```diff
@@ -0,0 +1,274 @@
+spec:
+  type: Spec
+  documentation_url: https://docs.airbyte.com/integrations/sources/bamboo-hr
+  connection_specification:
+    type: object
+    $schema: http://json-schema.org/draft-07/schema#
+    required:
+      - api_key
+      - subdomain
+      - custom_reports_fields
+    properties:
+      api_key:
+        type: string
+        description: Api key of bamboo hr
+        order: 0
+        title: api_key
+        airbyte_secret: true
+      subdomain:
+        type: string
+        description: Sub Domain of bamboo hr
+        order: 2
+        title: subdomain
+        description: mycompany
+      custom_reports_fields:
+        type: array
+        order: 3
+        title: custom_reports_fields
+        default:
+          - bestEmail
+          - firstName
+          - lastName
+          - age
+          - bonusAmount
+          - bonusDate
+          - city
+          - commissionAmount
+          - commisionDate
+          - country
+          - createdByUserId
+          - dateOfBirth
+          - department
+          - division
+          - eeo
+          - employeeNumber
+          - employmentHistoryStatus
+          - exempt
+          - fullName1
+          - gender
+          - hireDate
+          - originalHireDate
+          - homeEmail
+          - homePhone
+          - isPhotoUploaded
+          - jobTitle
+          - lastChanged
+          - location
+          - maritalStatus
+          - middleName
+          - mobilePhone
+          - nationalId
+          - nationality
+          - nin
+          - payChangeReason
+          - payGroup
+          - payGroupId
+          - payRate
+          - payRateEffectiveDate
+          - payType
+          - paySchedule
+          - payScheduleId
+          - payFrequency
+          - ssn
+          - sin
+          - standardHoursPerWeek
+          - state
+          - stateCode
+          - status
+          - supervisor
+          - supervisorId
+          - supervisorEId
+          - supervisorEmail
+          - terminationDate
+          - workEmail
+          - workPhone
+          - zipcode
+        description: List of fields to include in custom reports
+    additionalProperties: true
+type: DeclarativeSource
+check:
+  type: CheckStream
+  stream_names:
+    - meta_fields_stream
+streams:
+  - name: employees_directory_stream
+    type: DeclarativeStream
+    retriever:
+      type: SimpleRetriever
+      paginator:
+        type: NoPagination
+      requester:
+        path: employees/directory
+        type: HttpRequester
+        url_base: https://api.bamboohr.com/api/gateway.php/{{ config['subdomain'] }}/v1/
+        http_method: GET
+        authenticator:
+          type: BasicHttpAuthenticator
+          password: ''
+          username: '{{ config[''api_key''] }}'
```

Comment:
Can you confirm this works? Previous used some encoding https://github.com/airbytehq/airbyte/blob/b78ad4e9d905f2f02631f7897acabf6441375032/airbyte-integrations/connectors/source-bamboo-hr/source_bamboo_hr/source.py#L126-L135

Reply:
yes it works.
I have tested it